manager/manager/typeresolver.go

```diff
@@ -129,32 +139,41 @@ func (tr *typeResolver) ResolveTypes(config *Configuration, imports []*ImportFil
 		}
 
 		if sch != "" {
-			var s Schema
+			var s common.Schema
 			if err := yaml.Unmarshal([]byte(sch), &s); err != nil {
 				return nil, resolverError(config, err)
 			}
 			// Here we handle any nested imports in the schema we've just fetched.
 			for _, v := range s.Imports {
-				i := &ImportFile{Name: v.Name}
+				i := &common.ImportFile{Name: v.Name}
 				var existingSchema string
-				if len(fetched[v.Path]) == 0 {
+				u, conversionErr := tr.MapFetchableURL(v.Path)
```

Comment:
This whole code is likely broken (i.e., path is most likely a relative path, so won't work like this). I don't know if you want to fix it here or wait, I was likely going to be coming around to fixing it soon.